test/test_jit.py

```diff
@@ -1079,6 +1079,29 @@ def test_fn(ten, mask):
         self.assertEqual(test_fn(ten, mask), traced_test_fn(ten, mask))
 
 
+class TestBatched(TestCase):
+    def assertEqual(self, xs, ybs, prec=None, message='', allow_inf=False):
```

Comment:
It think it would be easier to understand these tests if assertEqual didn't attempt to expand a batched tensor automatically.  Then we can use the standard assertEquals:

```
self.assertEqual(xs, batch.examples())
```